api/client.js

```diff
@@ -182,6 +182,19 @@ function connectAndLogin(url, credentials, options, callback) {
 }
 
 
+/**
+  Returns a url that is to be used to connect to a supplied model uuid on the
+  supplied controller url.
+  @param {String} controllerURL The url that's used to connect to the controller.
```

Comment:
Looking at the code, this does not seem technically a URL. Maybe `controllerHost` is more adequate?

Reply:
Yeah I was thinking the same thing :) Updated